fava/static/package.json

```diff
@@ -30,6 +30,7 @@
     "jquery-dragster": "github:catmanjan/jquery-dragster",
     "jquery-query-object": "^2.2.3",
     "jquery-stupid-table": "github:medsabir/Stupid-Table-Plugin#feat-npm-publish",
-    "mousetrap": "^1.5.3"
+    "mousetrap": "^1.5.3",
+    "query-string": "^4.1.0"
```

Comment:
Ahh, got it. Didn't see that, will fix it/remove one of the two. Do you have a preference for which to remove?